salvo/tools/.style.yapf

```diff
@@ -0,0 +1,6 @@
+# The Google Python styles can be found here: https://github.com/google/styleguide/blob/gh-pages/pyguide.md
+# TODO: Look into enforcing single vs double quote.
```

Comment:
Glad to see we are planning to follow the linked style guide and bonus points for including automated checks using flake8. This will make it much easier for us to maintain the code going forward.